docs/en/writing-running-appium/ios/ios-xctest-pasteboard.md

```diff
@@ -3,8 +3,7 @@
 There is a possibility in Appium to set the content of the iOS Simulator pasteboard
 and read the content from there if needed. Each Simulator maintains its own pasteboard.
 This feature is only available since Xcode SDK 8.1.
-
-It might work when WebDriverAgentRunner is in [foreground on real devices](https://github.com/appium/WebDriverAgent/issues/330).
+Unfortunately, real devices don't provide such features.
```

Comment:
I would rather say: On real devices this functionality is available with several security limitations. Check `link to set/get pasteboard commands` for more details